src/AndroidDebugLauncher/StringExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+
+namespace AndroidDebugLauncher
+{
+    internal static class StringExtensions
+    {
+        public static IEnumerable<string> GetLines(this string content)
```

Comment:
Because this handles the various line endings for us already.

---
In reply to: [59105891](https://github.com/Microsoft/MIEngine/pull/267#discussion_r59105891) [](ancestors = 59105891)